daemon/k8s_watcher.go

```diff
@@ -1912,6 +1912,19 @@ func (d *Daemon) updateK8sNodeTunneling(k8sNodeOld, k8sNodeNew *v1.Node) error {
 	if !selfOwned {
 		return fmt.Errorf("ipcache entry owned by kvstore or agent")
 	}
+
+	routeTypes := node.TunnelRoute
+
+	// Add IPv6 routing only in non encap. With encap we do it with bpf tunnel
+	// FIXME create a function to know on which mode is the daemon running on
+	var ownAddr net.IP
+	if option.Config.AutoIPv6NodeRoutes && option.Config.Device != "undefined" {
```

Comment:
That `undefined` should be a const no?

Reply:
and also test for the string being empty?